lib/pages/chat_draft/draft_chat_empty_widget.dart

```diff
@@ -1,10 +1,10 @@
 import 'package:flutter/material.dart';
 import 'package:flutter_gen/gen_l10n/l10n.dart';
 
-class DirectDraftChatView extends StatelessWidget {
+class DraftChatEmpty extends StatelessWidget {
   final void Function()? onTap;
 
-  const DirectDraftChatView({
+  const DraftChatEmpty({
```

Comment:
`DraftChatEmptyView`

Reply:
~`DraftChatEmptyView`~ -> `DraftChatEmptyWidget` because it's a button and not a view